# no-noop

Disallows the [`$.noop`](https://api.jquery.com/jQuery.noop/) property. Prefer `function(){}`.

## Rule details

‚ùå Examples of **incorrect** code:
```js
$.noop;
$.noop();
```

‚úîÔ∏è Examples of **correct** code:
```js
foo.noop;
foo.noop();
foo.noop( bar );
$.noOp;
```

üîß Examples of code **fixed** by using  `--fix`:
```js
$.noop;   /* ‚Üí */ ( function () {} );
$.noop(); /* ‚Üí */ ( function () {}() );
```

## Resources

* [Rule source](/src/rules/no-noop.js)
* [Test source](/src/tests/no-noop.js)
