# no-noop

Disallows the [`$.noop`](https://api.jquery.com/jQuery.noop/) property. Prefer `function(){}`.

üîß The `--fix` option on the [command line](https://eslint.org/docs/user-guide/command-line-interface#fixing-problems) can automatically fix some of the problems reported by this rule.

## Rule details

‚ùå Examples of **incorrect** code:
```js
$.noop;
$.noop();
```

‚úîÔ∏è Examples of **correct** code:
```js
foo.noop;
foo.noop();
foo.noop( bar );
$.noOp;
```

üîß Examples of code **fixed** by this rule:
```js
$.noop;   /* ‚Üí */ ( function () {} );
$.noop(); /* ‚Üí */ ( function () {}() );
```

## Resources

* [Rule source](/src/rules/no-noop.js)
* [Test source](/src/tests/no-noop.js)
