# no-error-shorthand

Disallows the [`.error`](https://api.jquery.com/error/) method. Prefer `.on` or `.trigger`.

‚öôÔ∏è This rule is enabled in `plugin:no-jquery/deprecated-1.8`.

## Rule details

‚ùå Examples of **incorrect** code:
```js
$( 'div' ).error( handler );
$div.error();
$( 'div' ).first().error();
$( 'div' ).append( $( 'input' ).error() );
```

‚úîÔ∏è Examples of **correct** code:
```js
error();
[].error();
div.error();
div.error;
$.error();
```

üîß Examples of code **fixed** by using  `--fix`:
```js
$( 'div' ).error( handler ); /* ‚Üí */ $( 'div' ).on( 'error', handler );
$div.error();                /* ‚Üí */ $div.trigger( 'error' );
```

## Resources

* [Rule source](/src/rules/no-error-shorthand.js)
* [Test source](/src/tests/no-error-shorthand.js)
