# no-is-function

Disallows the [`$.isFunction`](https://api.jquery.com/jQuery.isFunction/) utility. Prefer `typeof`.

‚öôÔ∏è This rule is enabled in `plugin:no-jquery/deprecated-3.3`.

## Rule details

‚ùå Examples of **incorrect** code:
```js
$.isFunction( expression( arg ) );
if ( $.isFunction( fn ) ) { g(); }
```

‚úîÔ∏è Examples of **correct** code:
```js
isFunction();
myClass.isFunction();
$div.isFunction();
```

üîß Examples of code **fixed** by using  `--fix`:
```js
$.isFunction( expression( arg ) ); /* ‚Üí */ typeof expression( arg ) === 'function';
if ( $.isFunction( fn ) ) { g(); } /* ‚Üí */ if ( typeof fn === 'function' ) { g(); }
```

## Resources

* [Rule source](/src/rules/no-is-function.js)
* [Test source](/src/tests/no-is-function.js)
